src/generator/css/base.less

```diff
@@ -645,17 +645,37 @@ body .content .card-list {
 }
 
 .post-list {
-    @apply grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-8;
+    @apply grid ;
 
     .post {
-        @apply mb-16 align-text-bottom flex flex-col;
+        @apply mb-16  align-text-bottom grid grid-cols-1 md:grid-cols-10 gap-10 ;
+        
+        
     
         a.action {
-            @apply self-start mt-auto;
+            @apply self-start mt-auto float-left md:float-right;
+            
+            align-self:flex-end
+        }
+        .image {
+            @apply col-span-4  hidden md:block;
+            max-height: 40vh;
+         
+            img {
+                
+                height: 100%;
+                width: 100%;
+                object-fit: cover;
+            }
         }
-    
+        .body {
+            @apply col-span-6;
+        }
+
         .post-body {
             @apply mb-3;
+            width: 90%;
```

Comment:
Could you clarify the purpose of this declaraction?

Reply:
Removed width, had it before the column attributes, so the distribution of the text before was a bit off, now not necessary. The mb-3 is because the CAT button was too close to text and was almost touching the text.